chef/data_bags/crowbar/bc-template-nova_dashboard.json

```diff
@@ -36,7 +44,7 @@
     "nova_dashboard": {
       "crowbar-revision": 0,
       "crowbar-applied": false,
-      "schema-revision": 30,
+      "schema-revision": 32,
```

Comment:
In preparation to https://github.com/crowbar/barclamp-nova_dashboard/pull/216 that have 31